lua/conform/formatters/ktlint.lua

```diff
@@ -0,0 +1,10 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://ktlint.github.io/",
+    description = "An anti-bikeshedding Kotlin linter with built-in formatter.",
+  },
+  command = "ktlint",
+  args = { "--format", "--stdin", "**/*.kt", "**/*.kts" },
+  stdin = true,
```

Comment:
this is the default and can be omitted

Reply:
Usually I like to be explicit, do you want me to delete this line?